kubetest/gke.go

```diff
@@ -514,6 +532,11 @@ func (g *gkeDeployer) Down() error {
 			g.location)...))
 	errFirewall := finishRunning(exec.Command("gcloud", "compute", "firewall-rules", "delete", "-q", firewall,
 		"--project="+g.project))
+	var errSubnet error
+	if g.subnetwork != "" {
+		errSubnet := finishRunning(exec.Command("gcloud", "compute", "networks", "subnets", "delete", "-q", g.subnetwork,
+			g.subnetworkRegion, "--project="+g.project))
```

Comment:
--region=g.subnetworkRegion?

Reply:
Nope, it's already the whole thing. See the description of the flag from which it's computed.

Reply:
got it 👍